app/views/schools/participants/show.html.erb

```diff
@@ -178,21 +178,55 @@
       <% end %>
     </dl>
 
-    <% if @ects.present? %>
-      <h3 class="govuk-heading-m"><%= @induction_record.participant_full_name.split(" ").first %> is mentoring these
-        ECTs</h3>
-
-      <%= govuk_summary_list(classes: ["govuk-!-margin-bottom-7"]) do |list| %>
-        <% @ects.each do |ect| %>
-          <% list.with_row do |row| %>
-            <% row.with_key(text: ect.full_name,
-                       classes: ["govuk-!-font-weight-regular"]) %>
-            <% if @mentors_added &&
-                  ect.participant_profile.policy_class.new(current_user, ect.participant_profile).update_mentor? %>
-              <% row.with_action(text: "Change",
-                            visually_hidden_text: "mentor",
-                            href: school_participant_edit_mentor_path(participant_id: ect.participant_profile_id,
-                                                                      from_mentor: @profile.id)) %>
+    <% if @profile.mentor? %>
+      <h3 class="govuk-heading-m">Mentoring</h3>
+
+      <%= govuk_summary_list(classes: ["govuk-!-margin-bottom-7"], actions: false) do |slist| %>
+        <% slist.with_row do |row| %>
+          <% row.with_key do %>
+            Currently<br>mentoring
```

Comment:
Can we please remove the `<br>` so the text renders as a single line?
```suggestion
            Currently mentoring
```

Reply:
I added it because that is what seems to be in the picture attached to the ticket by @frankieroberto. 
We can confirm with him we want to display this in a single line.

Reply:
@ltello hiya - let's remove it please. It was probably only split across two lines in the screenshot because the browser window was a bit narrower.